samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/facade/QuestionPoolFacadeQueries.java

```diff
@@ -1417,11 +1417,18 @@ public Object doInHibernate(Session session) throws HibernateException, SQLExcep
 	  return count;
   }
   
+  /**
+   * Fetch a HashMap of question pool ids and counts for all pools that a user has access to.
+   * We inner join the QuestionPoolAccessData table because the user may have access to pools
+   * that are being shared by other users. We can't simply look for the ownerId on QuestionPoolData.
+   * This was originally written for SAM-2463 to speed up these counts. 
+   * @param agentId Sakai internal user id. Most likely the currently logged in user
```

Comment:
I think you should have said `@param agentId 87.2% chance of being the currently logged in user.` :stuck_out_tongue_winking_eye: